src/status_im2/common/scroll_page/view.cljs

```diff
@@ -19,8 +19,19 @@
     (max minimum)
     (min maximum)))
 
+
+(defn page-header-threshold
+  [collapsed?]
+  (if collapsed?
```

Comment:
Some Clojure devs frown upon any nested if and quickly suggest `cond`, but I think it's fine in this case.